### PR TITLE
SeverServices_UserOperationsWithMoney

### PR DESCRIPTION
**Pull request by ifelseelif**

### CODE REVIEW

File: Clients/GUI/Pages/UserTransactions.razor [hunk 1]
```diff
@@ -0,0 +1,81 @@
+﻿@page "/transactions"
```
Comment (nmamro) [inline Clients/GUI/Pages/UserTransactions.razor]: Change the design so it's under menu **UserInfo  Transactions**, so the choice does not disappear after getting to transactions

**Review by nmamro — COMMENTED** [review]
Ok

**Review by strannick-rodion — APPROVED** [review]

**Review by Cerberous007 — COMMENTED** [review]
Do we need all this js files??

File: Services/DataBaseService/Repositories/TradeRepository.cs [hunk 2]
```diff
@@ -166,5 +167,36 @@ public void UpdateUserBalance(UserBalance userBalance)
...
+                foreach (DbTransaction transaction in dbUserTransactions)
+                {
+                    userTransactions.Add(mapper.MapToTransaction(transaction));
+                }
+
+                return userTransactions;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```
Comment (Cerberous007) [inline Services/DataBaseService/Repositories/TradeRepository.cs]: Why not logging this exception?

File: Services/OperationService/Commands/GetUserTransactionsCommand.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        public async Task<IEnumerable<Transaction>> Execute(GetUserTransactionsRequest request)
+        {
+            var getUserTransactions = new GetUserTransactions
+            {
+                UserId = request.UserId
+            };
+
+            var result = await GetUserTransactions(getUserTransactions);
+
+            logger.LogInformation($"Transactions of user{request.UserId} taken successfully");
```
Comment (Cerberous007) [inline Services/OperationService/Commands/GetUserTransactionsCommand.cs:47]: If we throw exception in DataBase, this line also will be execute? correct me if I`m wrong

Comment (dTarasova) [reply Services/OperationService/Commands/GetUserTransactionsCommand.cs:47]: No, I think it wouldn't, it doesn't make full path of request

Comment (Cerberous007) [reply Services/OperationService/Commands/GetUserTransactionsCommand.cs:47]: Yes, you`re right

**Review by dTarasova — COMMENTED** [review]
Seems good. Suggestions to add pagination and search by fields, but not necessary

File: Services/DataBaseService/Repositories/TradeRepository.cs (hunk 2 above)
Comment (ifelseelif) [reply Services/DataBaseService/Repositories/TradeRepository.cs]: it is my mistake, I forgot to delete

File: Clients/GUI/Pages/UserTransactions.razor (hunk 1 above)
Comment (ifelseelif) [reply Clients/GUI/Pages/UserTransactions.razor]: Transactions moved in UserInfo

**Review by nmamro — APPROVED** [review]

**Review by denis-kriv — APPROVED** [review]

**Review by Cerberous007 — APPROVED** [review]

**Pull request merged**